packages/presentation/lib/ui/movie_details/bloc/movie_details_bloc.dart

```diff
@@ -40,6 +45,14 @@ class _MovieDetailsBlocImpl
     _updateData();
   }
 
+  @override
+  void share(BuildContext context) {
```

Comment:
https://stackoverflow.com/questions/58505388/is-it-ok-to-pass-buildcontext-to-bloc